content/posts/2024/2024-10-f2f63-seattle/forum/index.md

```diff
@@ -0,0 +1,438 @@
+* **Consensus = Dimitris and Ben W will look into this and come up with a proposed solution**
+
+### Mozilla Root Program Update  
+
+**Presenter:** Ben Wilson (Mozilla)  
+**Minutes:** Arvid  
+**Presentation link:** [Mozilla Root Program update](5-October-2024-Mozilla-News.pdf)  
+**Discussion outside the presentation:**  
+      
+* CA Compliance
+  * Overview of types of incidents, notably are the 22 delayed revocation bugs  
+  * CA inclusion requests: no big change  
+  * Revisions to Mozilla Root Store Policy  
+    * 263 change: no blank secions  
+    * 270/271: consitency between policies, no intention to make it more strict, some   
+    * 275: minor, grater emphasis on period-of-time audits  
+    * 276: delayed revocation requirements. Ben has sent out an email about the policy, there is some time during day 2 of the F2F to go through in more depth. Two-part proposal: a) require more detail in incident reports b) delayed revocation will result in a 90-day certificate requirement for those domains/FQDN that were in scope of the delay  
+    * 278: certificate linting requirements  
+    * 279: phasing out of multi-purpose roots   
+    * 281: support P521  
+    * 283: new inclusion requests need to support automation  
+  * CRLite  
+    * Currently there are three revocation reasons. The idea is to focus on security-sensitive revocaton reasons instead or ordinary or administrative revocations.   
+    * Mozilla will seek CABF buy-in for their reason codes  
+    * Graphs of revocation reasons were presented, most common reasons are cessation, superseded, key_compromise and unspecified.  
+    * 67% of all revocations is superseded but poisoned by a single issuing CA.   
+    * Without godaddy 71% unspecified.   
+    * Graphs for the issuing CA with largest revocation sets were presented  
+    * Conclusion is that there needs to be better guidance and harmonisation on the revocation reason codes. It is suggested the TLS BR need to be modified / improved  
+  * Questions / discussion  
+    * Revocation reasons  
+      * Somebody asked why reason codes are needed at all: Ben responded they want to segregate security-sensitive from administrative revocations. What when customers submitted the wrong reason?   
+      * Trev mentioned that the only relevant one is key compromise, the others seem to be insignificant. Key compromise seems to be the only one consumers are not confused about.   
+      * LetsEncrypt remarked that 99% of revocations are at the request of the ACME client. Their distribution has very little to do with their policy, but more with ACME client implementation. For the unspecified, a huge chunk of that is their own testing.
+      * Trev remarked that CA can't educate users on the reasons.  
+      * Ryan Dickinson said if we want to group everything security sensitive under key\_compromise this might unintentionally blacklist certain keys for no further issuance even if that is not required  
```

Comment:
```suggestion
      * Ryan D said if we want to group everything security sensitive under key\_compromise this might unintentionally blocklist certain keys for no further issuance even if that is not required  
```